roles/validate_inventory/tasks/vendor.yml

```diff
@@ -9,8 +9,8 @@
   loop: "{{ groups['nodes'] }}"
 
 - name: "Check vendors role exists"
-  include_role: 
-    name: "vendors/{{ hostvars[item]['vendor'] | lower }}"
+  include_role:
```

Comment:
Could you also use FQCN in the include_role? also the following files don't have FQCN in the include_role let's change them too

roles/generate_discovery_iso/tasks/static.yml
roles/add_day2_node/tasks/main.yml

Thanks

Reply:
all roles coming from crucible need to be audited for FQCN, but I'm not doing that just now. I have changed the files you mentioned, but let's hold off on the rest for another PR.

Reply:
Thanks! I was referring only to the include_role part :smile: but I see you have changed them all, sorry if I was not clear.